hack/test-update-storage-objects.sh

```diff
@@ -114,12 +117,16 @@ ${UPDATE_ETCD_OBJECTS_SCRIPT}
 killApiServer
 
 
+
 #######################################################
 # Step 3 : Start a server which supports only the new api version.
 #######################################################
 
 KUBE_API_VERSIONS="${KUBE_NEW_API_VERSION}"
 RUNTIME_CONFIG="api/all=false,api/${KUBE_NEW_API_VERSION}=true"
+
+# This seems to reduce flakiness.
+sleep 1
```

Comment:
Yes, even with the wait.

I can add the extra sleeps, but in my experience this particular spot is where the sleep was needed.  Since it's a kludge (I don't consider the wait a kludge, though), I prefer the minimal kludging necessary.  WDYT?